examples/multimodal/Multimodal.ipynb

```diff
@@ -0,0 +1,219 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d4073749",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from gpt_index import SimpleDirectoryReader, GPTSimpleVectorIndex\n",
+    "from gpt_index.readers.file.base import (\n",
+    "    DEFAULT_FILE_EXTRACTOR, \n",
+    "    ImageParser,\n",
+    ")\n",
+    "from gpt_index.response.notebook_utils import (\n",
+    "    display_response, \n",
+    "    display_image,\n",
+    ")\n",
+    "from gpt_index.indices.query.query_transform.base import (\n",
+    "    ImageOutputQueryTransform,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a4b74e87",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "image_parser = ImageParser(keep_image=True, parse_text=True)\n",
```

Comment:
would be nice to annotate each cell + lines within each cell on what each thing is doing. can be easy for users to get lost

Reply:
also might actually be good to check in the outputs in this case (more for promotion purposes than good git practices, since it's such a visual example)

Reply:
revised.